### DISCUSSION TITLE
Add support for `OR` keywords filtering in projects

### DISCUSSION THREAD

**Discussion by marina-chibizova** [community/community#21244] [Projects and Issues]
### Original request
I cannot filter for any match of multiple keywords (an OR query) in a project board. I read the [docs](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/filtering-projects), the comma is supposed to work, and it works for labels or other fields. My issue is with the keywords. One keyword is filtered fine, but a comma just adds all the other issues from the project and doesn't serve as OR.

### Support response

> Currently, we do not support OR filtering across different fields or between keywords right now.
> We do have improvements for filtering behaviour on our radar though.

This function is really needed for more flexible use of views in beta projects

**Comment by rileybroughten** [comment]
👋  Thank you for your feedback! We know more advanced `or` filtering is something users need and it is on our backlog 👍

**Reply by dgteixeira** [reply]
@rileybroughten, can you share any specific timeline for this implementation?
Thanks in advance!

**Reply by rileybroughten** [reply]
Hello @dgteixeira! 👋 We don't have an exact timing of when this will be supported, but we know it is something users need and it is on our backlog! We will keep you posted once we have more details on when this will be available. 👍

**Reply by marina-chibizova** [reply]
@rileybroughten no updates on the timeline?

**Reply by rileybroughten** [reply]
@marina-chibizova no updates on the timing as we are looking how to improve the filtering on Projects as a whole to make it consistent and easy to use, I'll post an update here when we have more details!
